app/lib/features/news/pages/add_news_page.dart

```diff
@@ -269,32 +301,23 @@ class _AddNewsState extends ConsumerState<AddNewsPage> {
   }
 
   Widget slideTextPostUI(BuildContext context) {
-    textController.text = selectedNewsPost!.text ?? '';
+    final backgroundColor = ref.watch(
+      newsStateProvider.select((i) => i.currentNewsSlide?.backgroundColor),
+    );
     return Container(
-      padding: const EdgeInsets.symmetric(horizontal: 5.0),
+      padding: const EdgeInsets.all(5.0),
```

Comment:
On the frontend this has now `20` ... should this also be `20` then?

Reply:
Yes we can make 20.